### DISCUSSION TITLE
Upload a Markdown File to Post

### DISCUSSION THREAD

**Discussion by erinb223** [forem/forem#17727] [Feature Requests]
[This request](https://twitter.com/_romeopeter/status/1527338048622452740) came in from user @_romeopeter on Twitter: "It’ll be great if writers could upload a markdown file article to DEV instead of having to type it in always. Further changes could then be made in the WYSIWYG editor."

I think this is a great idea, and I can see it helping authors save time and effort as they distribute content across platforms. I've invited Romeo to add more details here as well.

**Comment by romeopeter** [comment]
Thank you @erinb223 for the call.
The DEV online editor is great but most authors write offline. It's straining to have to copy what you've written already into the editor. A better option would be to upload the markdown file of the already written article into the editor and then make further changes with the WYSIWYG to the article if needs be. It saves time!
